osbs/constants.py

```diff
@@ -89,3 +90,5 @@
 
 BUILD_TYPE_ORCHESTRATOR = object()
 BUILD_TYPE_WORKER = object()
+
+ISOLATED_RELEASE_FORMAT = re.compile('^\d+\.\d+(\..+)?$')
```

Comment:
Use 'r' prefix to calm landscape.io: `re.compile(r'^\d+\.\d+(\..+)?$')`

Reply:
Done.